### DISCUSSION TITLE
dank grinder

### DISCUSSION THREAD

**Discussion by georgeous11** [BridgeSenseMC/Dank-Memer-Grinder#228] [General]
Am I able to use dank grinder with your new commits?

**Comment by MEZBATHEKING** [comment]
I completely understand your eagerness, but unfortunately, there is no set time frame for when Dank Memer Grinder will be available for everyone to use. However, based on current progress, the community, along with myself, expects everything to be in order within the next 1-2 weeks. Your patience is greatly appreciated.

**Reply by georgeous11** [reply]
awesome! thank you so much!

**Comment by georgeous11** [comment]
Will there be an announcement when it is ready?
